modules/victoria-metrics/templates/values.yaml

```diff
@@ -233,6 +233,9 @@ vmsingle:
       resources:
         requests:
           storage: 20Gi
+    nodeSelector: {
+      spotinst.io/node-lifecycle: "od"
+    }
```

Comment:
These are not new changes (Git is showing them in the diff for some reason). Please ignore changes here.